documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -175,3 +175,49 @@ strimzi-topic-operator  1/1    1           1
 +
 `READY` shows the number of replicas that are ready/expected.
 The deployment is successful when the `AVAILABLE` output shows `1`.
+
+== (Preview) Deploying the standalone Topic Operator for unidirectional topic management
+
+Unidirectional topic management maintains topics solely through `KafkaTopic` resources.
+For more information on unidirectional topic management, see xref:ref-operator-topic-{context}[].
+
+If you want to try the preview of unidirectional topic management, follow these steps to deploy the standalone Topic Operator:
+
+. Undeploy the current standalone Topic Operator.
++
+Retain the `KafkaTopic` resources, which are picked up by the Topic Operator when it is deployed again.
+
+. Edit the `Deployment` configuration for the standalone Topic Operator to remove any ZooKeeper-related environment variables:
++
+* `STRIMZI_ZOOKEEPER_CONNECT`
+* `STRIMZI_ZOOKEEPER_SESSION_TIMEOUT_MS`
+* `TC_ZK_CONNECTION_TIMEOUT_MS`
+* `STRIMZI_USE_ZOOKEEPER_TOPIC_STORE`
++
+Unidirectional topic management does not use ZooKeeper.
++
+Other unused environment variables that can be removed if present:
++
+* `STRIMZI_REASSIGN_THROTTLE`
+* `STRIMZI_REASSIGN_VERIFY_INTERVAL_MS`
+* `STRIMZI_TOPIC_METADATA_MAX_ATTEMPTS`
+* `STRIMZI_TOPICS_PATH`
+* `STRIMZI_STORE_TOPIC`
+* `STRIMZI_STORE_NAME`
+* `STRIMZI_APPLICATION_ID`
+* `STRIMZI_STALE_RESULT_TIMEOUT_MS`
+
+. (Optional) Set the `STRIMZI_USE_FINALIZERS` environment variable to `true`:
++
+.Additional configuration for unidirectional topic management
+[source,shell,subs=+quotes]
+----
+# ...
+env:
+  - name: STRIMZI_USE_FINALIZERS
+    value: "true"
+----
++
+Use this environment variable if you want to use finalizers to control xref:con-deleting-managed-topics-{context}[topic deletion].
+
+. Deploy the standalone Topic Operator in the standard way.
```

Comment:
I assume that there will be also some change to the startup script needed? Or some additional environment variable?

Reply:
I'm assuming the zookeeper variables make the difference, as with change in the [topic_operator_run.sh](https://github.com/strimzi/strimzi-kafka-operator/pull/8627/files#diff-ea49bbb821c918d163e90ec3da950dffae099d3871e3983d7792a2f45584bf96):

_If ZooKeeper connection details are provided then use the old bidirectional TO
Otherwise use the new unidirectional TO_